doc/developer/building-libyang.rst

```diff
@@ -5,12 +5,13 @@ library.
 
 **Option 1: Binary Install**
 
-The FRR project builds binary ``libyang`` packages, which we offer for download
-`here <https://ci1.netdef.org/browse/LIBYANG-YANGRELEASE/latestSuccessful/artifact>`_.
+The FRR project builds some binary ``libyang`` packages.
+
+RPM packages are at our `RPM repository <rpm.frrouting.org>`_
 
 .. warning::
 
-   ``libyang`` version 0.16.105 or newer is required to build FRR.
+   ``libyang`` version 1.0.0 or newer is required to build FRR.
```

Comment:
We require version >= 1.0.184. Not sure if we need this change now since we haven't really migrated to 1.x yet.

Reply:
OK